src/auth/auth-api-request.ts

```diff
@@ -633,10 +669,10 @@ const LIST_INBOUND_SAML_CONFIGS = new ApiSettings('/inboundSamlConfigs', 'GET')
 /**
  * Class that provides the mechanism to send requests to the Firebase Auth backend endpoints.
  */
-export class FirebaseAuthRequestHandler {
-  private readonly httpClient: AuthorizedHttpClient;
-  private readonly authUrlBuilder: AuthResourceUrlBuilder;
-  private readonly projectConfigUrlBuilder: AuthResourceUrlBuilder;
+export class BaseFirebaseAuthRequestHandler {
```

Comment:
Can we make this class abstract and rename it to `AbstractAuthRequestHandler`? 

`getAuthUrlBuilder()` and `getProjectConfigUrlBuilder()` could be abstract methods of the class.

Reply:
Done.